equivalence_test.go

```diff
@@ -37,9 +37,6 @@ import (
 )
 
 func TestStatsAndMetricsEquivalence(t *testing.T) {
-	_, _, stop := createFakeServer(t)
```

Comment:
Why remove this?

Reply:
Good question, why keep it? Seems that we just defer the stop but definitely we don't use it because we don't save any value